apps/google-analytics-4/frontend/.env

```diff
@@ -1,3 +1,4 @@
 REACT_APP_BACKEND_API_URL=https://google-analytics-4.ctfapps.net
 REACT_APP_VERSION=$npm_package_version
-REACT_APP_SENTRY_DSN=https://b8f524eae7c446fb8071476431426640@o2239.ingest.sentry.io/4504725335834624
\ No newline at end of file
+REACT_APP_SENTRY_DSN=https://b8f524eae7c446fb8071476431426640@o2239.ingest.sentry.io/4504725335834624
+REACT_APP_SEGMENT_WRITE_KEY=jGZHaYldsH12362MNsO2mT0BEvV2Hi6W
```

Comment:
production writeKey (source: https://app.segment.com/contentful/sources/google_analytics_4_app/overview)

Reply:
added to circle CI env vars ☝️